src/worker/queues/nonceHealthCheckQueue.ts

```diff
@@ -0,0 +1,14 @@
+import { Queue } from "bullmq";
+import { redis } from "../../utils/redis/redis";
+import { defaultJobOptions } from "./queues";
+
+export class NonceHealthCheckQueue {
+  static q = new Queue<string>("nonceHealthCheck", {
```

Comment:
nit: The other queue names are kebab-cased. Could you make this kebab-case too? (There was no specific reason other than I just started them that way)